packages/sdk-snippets/tsup.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig(options => ({
+  ...options,
+
+  cjsInterop: true,
+  dts: true,
+  entry: ['src/index.ts'],
+  format: ['esm', 'cjs'],
+  shims: true,
+  silent: !options.watch,
+  sourcemap: true,
+  treeshake: true,
```

Comment:
i think the treeshaking flag has been problematic for us historically, see this PR: https://github.com/readmeio/fetch-har/pull/427

```suggestion
```